src/components/views/settings/tabs/user/PreferencesUserSettingsTab.tsx

```diff
@@ -254,6 +287,22 @@ export default class PreferencesUserSettingsTab extends React.Component<IProps,
                     </SettingsSubsection>
 
                     <SettingsSubsection heading={_t("settings|preferences|time_heading")}>
+                        <div className="mx_SettingsSubsection_dropdown">
+                            {_t("settings|preferences|user_timezone")}
+                            <Dropdown
```

Comment:
Can you use instead our [dropdown component from our design system](https://compound.element.io/?path=/docs/compound-web_dropdown--docs)

Reply:
Unless I'm mistaken the dropdown in question does not have a search and with over 400 elements I think the search is quite useful.